tests/classes.spec.ts

```diff
@@ -55,5 +55,22 @@ describe('classes', () => {
       expect(wrapper.classes('class-a')).toBe(true)
       expect(wrapper.classes('class-b')).toBe(false)
     })
+
+    it('returns error if the component does not have the class', () => {
```

Comment:
nit: `returns error if the component does not have the class` => `throws an error if called on an empty wrapper`

Reply:
Thank you. Fixed.